database/zynq7/mapping/devices.yaml

```diff
@@ -1,5 +1,5 @@
 # device to fabric mapping
```

Comment:
This file is still checked in

Reply:
Yes. I would keep them checked in because of two reasons. First, it tells which device is supported and therefore only parts for these devices get generated. Secondly, the device to fabric mapping seems to be a human thing which can be generated?
What do you think?

Reply:
Everything under database is usually seeded by bringing in `prjxray-db` and `download-latest-db.sh`, so having this generated file checked into `prjxray` doesn't make sense.  Please remove.